<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="coin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 52, 52);
font: 11pt;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="distance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54,54,54);
font: 11pt;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>341</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="restart">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 54, 54);
font: 16pt;</string>
     </property>
     <property name="text">
      <string>重新开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="return_main">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 54, 54);
font: 16pt;</string>
     </property>
     <property name="text">
      <string>返 回</string>
     </property>
    </widget>
    <widget class="QLabel" name="over_score">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 54, 54);
font: 15pt ;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="over_distance">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 54, 54);
font: 15pt ;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="game_over">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 54, 54);
font: 25pt ;</string>
     </property>
     <property name="text">
      <string>完全去世了。</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 54, 54);
font: 18pt;</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intro">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 54,54);
font: 18pt;</string>
    </property>
    <property name="text">
     <string>游戏介绍</string>
    </property>
   </widget>
   <widget class="QLabel" name="record">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54,54,54);
font: 11pt;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
